lib/eosknowledgeprivate/ekn-media-bin.c

```diff
@@ -1401,13 +1408,66 @@ ekn_media_bin_update_position (EknMediaBin *self)
   gtk_label_set_label (priv->audio_position_label, format_time (position));
 }
 
+static inline void
+log_fps (EknMediaBin *self)
+{
+  EknMediaBinPrivate *priv = EMB_PRIVATE (self);
+  GSource *source = g_main_current_source ();
```

Comment:
I'm not entirely sure it's a good idea to use the time of the current GSource, here. You're not in a GSource callback, you're in a tick callback provided by GtkWidget.